.github/workflows/end2end.yml

```diff
@@ -0,0 +1,62 @@
+name: End-to-end test of EasyBuild in different distros
+on: [push, pull_request]
+jobs:
+  build_publish:
+    name: End-to-end test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container:
+          - centos-7.9
+          - centos-8.4
+          - fedora-35
```

Comment:
For now, it works, and if it starts causing trouble it'll be very easy to kick it out from this workflow configuration.
So my vote goes to keeping it in, for now.

Testing in Fedora makes sense to me since it can act as a canary-in-the-coal-mine: if EasyBuild starts failing in Fedora, then that's something we should know ASAP, since that's going to trickly down into RHEL & co eventually...